tests/libstorage/pvc.go

```diff
@@ -50,6 +50,7 @@ const (
 
 func RenderPodWithPVC(name string, cmd []string, args []string, pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
 	volumeName := "disk0"
+	rootUser := int64(0)
```

Comment:
This can be PSA AFAIK
https://github.com/kubevirt/kubevirt/pull/8645/files#diff-b4e7f73de66144dcfc1208d87123e61db37341d94b02fd1ce0fcc89ae4a717c3R52

Reply:
Are you sure? When it's used inside `ChangeImgFilePermissionsToNonQEMU` it does things like`chown root:root ..` for which you have to be root.

Reply:
Yes, that's the only case where it needs root, so we can apply it just in `ChangeImgFilePermissionsToNonQEMU`
Other usages of RenderPodWithPVC do not require root
https://github.com/kubevirt/kubevirt/pull/8645/files#diff-847cd06c1bf6ef597ea96894f8b32ecb4d074ad1bc785b39871e11fb6ab71b3dR1321
(Ran CI on this in my draft, was green)